workshop/01_Navigation/02_Navigate_to_Declaration/2.1_Go_to_declaration.php

```diff
@@ -1,4 +1,5 @@
 <?php
+<?php
```

Comment:
Additional php opening tag. Sorry, haven't noticed in the previous pull request.